tox.ini

```diff
@@ -99,6 +91,15 @@ setenv =
     SUPERSET_CONFIG = tests.superset_test_config_sqllab_backend_persist
     SUPERSET_HOME = {envtmpdir}
 
+[testenv:docs]
+commands =
+    cp -r superset-frontend/images/ docs/_static/images/
```

Comment:
This copy statement should now be removed, as we've replaced it with a symlink (I forgot to do this fix in `tox.ini` as CI isn' t using this anymore)